Exiled.Permissions/Exiled.Permissions.csproj

```diff
@@ -27,6 +27,7 @@
     <Reference Include="Mirror" HintPath="$(EXILED_REFERENCES)\Mirror.dll" Private="false" />
     <Reference Include="UnityEngine.CoreModule" HintPath="$(EXILED_REFERENCES)\UnityEngine.CoreModule.dll" Private="false" />
     <Reference Include="NorthwoodLib" HintPath="$(EXILED_REFERENCES)\NorthwoodLib.dll" Private="false" />
+    <Reference Include="UnityEngine.PhysicsModule" HintPath="$(EXILED_REFERENCES)\UnityEngine.PhysicsModule.dll" Private="false" />
```

Comment:
why

Reply:
For the Player.Get(Collider) i thinks. Compiler give me a problem in build. Exiled.permission needs Physics, so i added it

Reply:
fair